Find a functionally equivalent code:android.widget.Switch.setTrackResource:COMMENT
Method Modifier: public      
Comment:/**
 * Set the drawable used for the track that the switch slides within.
 *
 * @param resId Resource ID of a track drawable
 *
 * @attr ref android.R.styleable#Switch_track
 */

@@ -1,3 +1,3 @@
 {
-    setTrackDrawable(getContext().getResources().getDrawable(resId));
+    setTrackDrawable(getContext().getDrawable(resId));
 }

